platforms/solana/src/address.ts

```diff
@@ -8,6 +8,7 @@ import {
 
 import { PublicKey } from '@solana/web3.js';
 import { AnySolanaAddress } from './types';
+import { SolanaPlatform } from '../dist/esm';
```

Comment:
This will probably break stuff in build

Reply:
Absolutely, I must have hit tab on the wrong line cause I did not mean to import from dist :|